com.yacreader.YACReader.yml

```diff
@@ -0,0 +1,123 @@
+app-id: com.yacreader.YACReader
+runtime: org.kde.Platform
+runtime-version: "5.15-22.08"
+sdk: org.kde.Sdk
+command: YACReader
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
```

Comment:
does it need network access?

Reply:
Yes, the library app has a builtin webserver which can be used to sync comics with a companion iOS app.